src/simtools/testing/validate_output.py

```diff
@@ -55,8 +55,12 @@ def validate_application_output(config):
         if "REFERENCE_OUTPUT_FILE" in integration_test:
             _validate_reference_output_file(config, integration_test)
 
+        if "OUTPUT_FILE_TESTS" in integration_test:
```

Comment:
This might be very pedantic, but what does `OUTPUT_FILE_TESTS` mean? Is it a file used to test the output? Or tests which check that the output file exists?

I think in both cases `test_output_file` would fit better, but I am not sure.

Reply:
It is tests results to output files. It appears clear to me, but if "TEST_OUTPUT_FILES" seems better, I change (really don't care).